pkgs/development/coq-modules/deriving/default.nix

```
@@ -7,12 +7,14 @@ mkCoqDerivation {
   owner = "arthuraa";
 
   inherit version;
-  defaultVersion = with lib.versions; lib.switch coq.coq-version [
-    { case = range "8.11" "8.18"; out = "0.1.1"; }
+  defaultVersion = with lib.versions; lib.switch [coq.coq-version ssreflect.version] [
+    { case = [(range "8.17" "8.18") (isGe "2.0.0")] ; out = "0.2.0"; }
+    { case = [(range "8.11" "8.18") (isLe "2.0.0")] ; out = "0.1.1"; }
```

If you use a list, you have to change `case` to `cases`. It's unfortunate that these errors are not caught automatically. I noticed because I tried opening a PR for testing on the Coq Nix Toolbox and it removed `deriving` from all CI workflows.

```suggestion
    { cases = [(range "8.17" "8.18") (isGe "2.0.0")] ; out = "0.2.0"; }
    { cases = [(range "8.11" "8.18") (isLe "2.0.0")] ; out = "0.1.1"; }
```

---

I'm testing the PR with this fix at: https://github.com/coq-community/coq-nix-toolbox/pull/168